src/types/Product.ts

```diff
@@ -0,0 +1,16 @@
+export type Product = {
+  id: number;
+  category: string;
+  itemId: string;
+  name: string;
+  fullPrice: number;
+  price: number;
+  screen: string;
+  capacity: string;
+  color: string;
+  ram: string;
+  year: number;
+  image: string;
+  createdAt?: string;
```

Comment:
createdAt and UpdatedAt fields should be removed, you won't get them from beckend